src/Features/VisualBasic/Portable/ConvertIfToSwitch/VisualBasicConvertIfToSwitchCodeRefactoringProvider.Analyzer.vb

```diff
@@ -29,6 +30,13 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.ConvertIfToSwitch
                         Return False
                 End Select
             End Function
+
+            Public Overrides Function CanImplicitlyConvert(semanticModel As SemanticModel, syntax As SyntaxNode, targetType As ITypeSymbol) As Boolean
+                Dim expressionSyntax = TryCast(syntax, ExpressionSyntax)
+
+                Return expressionSyntax IsNot Nothing AndAlso
+                    semanticModel.ClassifyConversion(expressionSyntax, targetType).IsWidening
```

Comment:
note: you can pull this up and just get the commonconversion in the base class.  that CommonConversion has a .IsImplicit which is what you want here.

Reply:
If you mean using the CommonConversion on the operator, that didn't work for all cases. If you mean getting the common conversion from the `ClassifyConversion` call, thats good to know, but since `ClassifyConversion` can't be called from the language agnostic side I don't think it saves much.

Reply:
There's no ClassifyCommonConversion?? :'(

Reply:
ahhh, its on Compilation. Thanks!

Reply:
.. and it works a bit differently. It works on types, which means dealing with conversions again. I'm gonna leave this 😛